chain/neutrino.go

```diff
@@ -0,0 +1,787 @@
+package chain
```

Comment:
We really should preserve the copyright from bchwallet since this isn't from the same repo